app/graphql/types/interfaces/rules_preload.rb

```diff
@@ -4,6 +4,7 @@
 # rule results, compliant,and references
 module RulesPreload
   include GraphQL::Schema::Interface
+  field_class GraphQL::Cache::Field
```

Comment:
Is this required for some reason? This module is implemented from classes which inherit from `BaseObject`, so I thought the field_class would be set already.